apps/passport-server/migrations/37_semphore_id.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE telegram_bot_conversations
```

Comment:
Adds semaphore id. Note that existing users on staging will need to be kicked out and rejoin, bc we don't have their semaphore IDs